apps/site/pages/docs/getting-started/onboarding.mdx

```diff
@@ -51,15 +51,15 @@ Now that you have an overview of the app, let's start the project.
 2. In the **Setup & Create Project** page, ensure to define the basic configuration for your store:
 
 #### CMS Integration
-The option available to use a CMS in a FastStore project is [VTEX Headless CMS](/tbd), a VTEX'S Content Management solution.
+The option available to use a CMS in a FastStore project is [VTEX Headless CMS](https://www.faststore.dev/tutorials/cms/0), a VTEX'S Content Management solution.
```

Comment:
I think that there's an extra `a` and the s after the apostrophe should be lower case

```suggestion
The option available to use a CMS in a FastStore project is [VTEX Headless CMS](https://www.faststore.dev/tutorials/cms/0), VTEX's Content Management solution.
```